app/assets/javascripts/discourse/tests/integration/components/slow-mode-info-test.js

```diff
@@ -0,0 +1,77 @@
+import componentTest, {
+  setupRenderingTest,
+} from "discourse/tests/helpers/component-test";
+import {
+  discourseModule,
+  queryAll,
+} from "discourse/tests/helpers/qunit-helpers";
+
+discourseModule("Integration | Component | slow-mode-info", function (hooks) {
+  setupRenderingTest(hooks);
+
+  componentTest("doesn't render if the topic is closed", {
+    template: "{{slow-mode-info topic=topic}}",
+
+    beforeEach() {
+      this.set("topic", { slow_mode_seconds: 3600, closed: true });
+    },
+
+    test(assert) {
+      assert.ok(queryAll(".slow-mode-heading").length === 0);
```

Comment:
If you are looking for the existence (or non-existence) of only one element, you should use the `exists` helper.

```suggestion
      assert.ok(!exists(".slow-mode-heading"), "it doesn't render the notice");
```